src/test/cypress/cypress/integration/TaskAdvanced.spec.js

```diff
@@ -0,0 +1,300 @@
+  });
+
+  describe('Send user card with question and answer it', function () {
+
+    it('Send User card with question from operator1_fr', () => {
+
+      opfab.loginWithUser('operator1_fr');
+      feed.checkNumberOfDisplayedCardsIs(0);
+      opfab.navigateToUserCard();
+      usercard.selectService('User card examples');
+      usercard.selectProcess('Message or question');
+      usercard.selectState('Message or question list');
+      cy.get('#message-select').click();
+      cy.get('#message-select').find('.vscomp-search-input').type('Confirmation the issues have been fixed');
+      cy.get('#message-select').find('.vscomp-option-text').eq(0).click();
+      usercard.previewThenSendCard();
+    })
+
+    it('Receive User card and answer it', () => {
+
+      opfab.loginWithUser('operator2_fr');
+      feed.checkNumberOfDisplayedCardsIs(1);
+      feed.openFirstCard();
+      cy.get("#message").contains("Please confirm the issues in your area have been fixed");
+      cy.get("#comment").type("My answer")
+      cy.get("#opfab-card-details-btn-response").click();
+      cy.get("#childs-div").contains("My answer");
+    })
+  })
+
+  describe('Send user card with question and answer it', function () {
+    before('Delete previous cards', function () {
+      script.deleteAllCards();
+      script.deleteAllArchivedCards();
+    });
+
+    it('Check task advanced card appears on the timeline and calendar', function () {
+
+      opfab.loginWithUser('operator1_fr');
+
+      setTimeLineDomain('7D');
+      checkHaveCircle(0);
+
+      // Sends 7 cards over the coming week
+      opfab.navigateToUserCard();
+      usercard.checkEmitterSelectDoesNotExist();
+      usercard.selectService('User card examples')
+      usercard.selectProcess('Task Advanced');
+      cy.waitDefaultTime();
+      cy.get('#time').type('00:00');
+      usercard.previewThenSendCard();
+
+      // Check first week has 7 cards
+      checkHaveCircle(7);
+
+      cy.get('#opfab-navbarContent').find('#opfab-calendar-menu').click();
+
+      // Check 7 cards are on the calendar screen with the default month view
+      cy.get('.opfab-calendar-event').should('have.length', 7);
+
+      // Delete the recurrent cards
+      opfab.navigateToFeed();
+      feed.openFirstCard();
+      feed.deleteCurrentCard();
+    });
+  });
+
+  describe('Send Task advanced user card, open the card and edit it', function () {
+
+    before('Delete previous cards', function () {
+      script.deleteAllCards();
+      script.deleteAllArchivedCards();
+    });
+
+    it('Send Task advanced user card for daily frequency from operator1_fr', () => {
+
+      opfab.loginWithUser('operator1_fr');
+      feed.checkNumberOfDisplayedCardsIs(0);
+      opfab.navigateToUserCard();
+      usercard.selectService('User card examples');
+      usercard.selectProcess('Task Advanced');
+      cy.get('#taskDescription').type('Test task for daily frequency');
+      cy.get('#Monday').click({force: true});
```

Comment:
{force:true} is necessary here because the input html field (checkbox)  is not visible and replace with a custom checkbox so when selected the input cypress consider it as hidden and failed
 
To solve that pb , you need to click on  the label :

```
<label class="opfab-checkbox" style="padding-left:25px"> ${opfab.utils.getTranslation("shared.calendar.monday")} <input type="checkbox" id="Monday">   <span class="opfab-checkbox-checkmark"> </span>   </label>
```

To do that you can do for example : 

```
cy.get('.opfab-checkbox').contains('Monday').click();
```
code seen in src/test/cypress/cypress/support/activityAreaCommands.js

So you should be able to remove all force:true 

It is better to not use force:true to avoid having a passing test whereas checkbox are not visible

Reply:
Done.